WORKSPACE

```diff
@@ -885,3 +885,21 @@ nixpkgs_package(
     nix_file_deps = common_nix_file_deps,
     repositories = dev_env_nix_repos,
 )
+
+nixpkgs_package(
+    name = "unixtools_script",
+    attribute_path = "unixtools.script",
+    nix_file = "//nix:bazel.nix",
+    nix_file_deps = common_nix_file_deps,
+    repositories = dev_env_nix_repos,
+) if not is_windows else None
+
+dev_env_tool(
```

Comment:
You shouldn’t need this one. `dev_env_tool` is really only for when we need it to work on Windows & Unix. You can reference the nixpkgs_package` directly.

Reply:
Removed!